### ISSUE TITLE
Invalid fix for D205+D400

### ISSUE THREAD

**Issue by serjflint** [charliermarsh/ruff#1039]
pyproject.toml

```toml
[tool.ruff]
extend-select = ["D"]
extend-ignore = [
    "D203",
    "D212",
    "D213",
    "D214",
    "D215",
    "D404",
    "D405",
    "D406",
    "D407",
    "D408",
    "D409",
    "D410",
    "D411",
    "D413",
    "D415",
    "D416",
    "D417",
]

```

test_ruff.py

```python
"""Stub."""


class Ruffed:
    """Stub."""

    def __call__(self, a, b) -> None:
        """
        :param a: table rows
        :param b: table rows
        """
        pass

```

Run ruff 0.0.156 with fix:

```bash
ruff test_ruff.py --fix
```

It adds one hundred empty lines after line 8

**Comment by charliermarsh** [comment]
One hundred empty lines! Not good! Will take a look, thanks for filing.

**Comment by charliermarsh** [comment]
(I’m guessing that the fixes are somehow conflicting and so it’s applying them repeatedly, up to 100 iterations.)

**Comment by charliermarsh** [comment]
This is resolved by https://github.com/charliermarsh/ruff/pull/1041, but the behavior probably won't be quite what you want. The "right" thing to do, AFAICT, is detect that `:param a: table rows` is the "summary line", and add a newline after that line. That is at least stable and consistent with the rule logic.
